ionicons/IonIcons.m

```diff
@@ -19,10 +19,33 @@ @implementation IonIcons
 + (UIFont*)fontWithSize:(CGFloat)size;
 {
     UIFont* font = [UIFont fontWithName:@"ionicons" size:size];
+    if (!font) {
+        [[self class] dynamicallyLoadFontNamed:@"ionicons"];
```

Comment:
Hey @sweetmandm thanks for the comments and rebase, the important part of the podspec change was really the resource bundle, the others were just while I was messing around. 

Pretty much the `if (!font)` check guarantees that the font is never loaded when it doesn't need to be. I am not positive if there is ever a time where the font will get deregistered during the app's lifetime and need to be loaded again. If not,  `dispatch_once` is a very nice solution as well!